src/OpenTelemetry/Metrics/Exemplar/ExemplarFilterType.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Metrics;
+
+#if EXPOSE_EXPERIMENTAL_FEATURES
+/// <summary>
+/// Defines the exemplar filters supported by the OpenTelemetry .NET Metrics
```

Comment:
```suggestion
/// Defines the exemplar filters supported by the Metrics
```

do we need to remind all that this is OpenTelemetry .NET 😆

Reply:
Brand reinforcement! 🤣 

Tweaked here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5404/commits/0884011a546a1121885848ce5db97317accfb2a1